packages/indexer/src/core/utils.ts

```diff
@@ -132,6 +132,37 @@ export function createLogger(
   });
 }
 
+function formatEVM(s: string) {
```

Comment:
What does this do? Might be good to include an example as a comment right above this function

Reply:
Actually, all of what's in there must have comments and the docs, but that comes right after this PR I believe.
This particular function is producing the evm's address formatted using [checksum](https://coincodex.com/article/2078/ethereum-address-checksum-explained/)